tests/GslCore.Tests/testPromTermLen.fs

```diff
@@ -0,0 +1,85 @@
+﻿module testPromTermLen
+/// Test #promoterlen #terminatorlen work
+
+open NUnit.Framework
+open constants
+open commonTypes
+open pragmaTypes
+open Amyris.ErrorHandling
+
+/// location of test gslc_lib fixtures (depends on .net tooling version)
+let testLibDir1 = @"../../../../TestGslcLib"
+let testLibDir2 = @"../../../../../TestGslcLib"
+
+[<TestFixture>]
+type TestPromTermLen() = 
+    let emptyPragmas = PragmaCollection(Map.empty)
+    do
+        // initialize pragmas
+        pragmaTypes.finalizePragmas []
+
+    let testLibDir = 
+        if System.IO.Directory.Exists testLibDir1 then testLibDir1  
+            else testLibDir2
+
+    let same context expected actual =
```

Comment:
Why this function instead of AssertEqual?  "context" should be apparent from the stack trace if a test fails.